packages/design-system/src/components/IdleTimeout/IdleTimeout.tsx

```diff
@@ -78,69 +73,128 @@ const defaultMessageFormatter = (timeTilTimeout: number): React.ReactNode => {
 };
 
 const IdleTimeout = ({
-  closeDialogText = 'Close',
   continueSessionText = 'Continue session',
   heading = 'Are you still there?',
   endSessionButtonText = 'Logout',
+  endSessionRedirectUrl = '/logout',
   formatMessage = defaultMessageFormatter,
-  onClose,
   onSessionContinue,
   onSessionForcedEnd,
   onTimeout,
   showSessionEndButton = false,
   timeToTimeout,
   timeToWarning = 5,
 }: IdleTimeoutProps) => {
-  const handleContinueSession = () => {
+  if (timeToWarning > timeToTimeout) {
+    console.error(
+      'Error in TimeoutManager component. `timeToWarning` is greater or equal to `timeToTimeout`'
+    );
+  }
+  // convert minutes to milliseconds
+  const msToTimeout = timeToTimeout * 60000;
+  const msToWarning = (timeToTimeout - timeToWarning) * 60000;
+  const [timeoutTimerId, setTimeoutTimer] = useState<NodeJS.Timeout>(null);
```

Comment:
I found this (https://stackoverflow.com/a/56239226) when searching for NodeJS.Timeout, seems like it may be better typing for this for platform independence?